Sources/SmileID/Classes/Metadata/NetworkMetadataProvider.swift

```diff
@@ -131,7 +131,6 @@ extension NetworkMetadataProvider: MetadataProvider {
                 date: $0.date
             )
         }
-        connectionEvents.removeAll()
```

Comment:
@tobitech any reason why we remove this? Shouldn't we reset the connectionEvents after upload?

Reply:
This particular line means connection events are removed before they are added to overall events that get sent to the API.